responders/src/UI/embc-responder/src/app/feature-components/search/essfile-dashboard/ess-file-overview/ess-file-overview.component.ts

```diff
@@ -12,6 +13,8 @@ import * as globalConst from '../../../../core/services/global-constants';
 export class EssFileOverviewComponent implements OnInit {
   essFile: EvacuationFileModel;
   animalCount = 0;
+  
+  IdentifyNeed = IdentifiedNeed
```

Comment:
not sure what this is for

Reply:
as far as this, all i can tell you is that this happens if it's not there 
![image](https://github.com/bcgov/embc-ess-mod/assets/47016475/81cd75d0-78f6-480f-9e5b-62a766b13df2)

Reply:
Because the component html is using it, to remove this you'll need to push the logic down to the code behind